ui/app/src/views/ViewProject.tsx

```diff
@@ -32,36 +36,69 @@ function DashboardPageInProject(props: RenderDashboardInProjectProperties) {
   const [openCreateDashboardDialogState, setOpenCreateDashboardDialogState] = useState(false);
 
   const { data } = useDashboardList(props.projectName);
-  if (data === undefined) {
-    return null;
-  }
 
   const handleDashboardCreation = function (name: string) {
     navigate(`/projects/${props.projectName}/dashboards/${name}/create`);
   };
 
   return (
-    <Paper>
-      <Box p={1}>
-        <Stack direction="row" alignItems="center" justifyContent="space-between">
-          <Stack direction="row" alignItems="center" gap={1} my={2}>
-            <ViewDashboard />
-            <Typography variant="h3">Dashboards</Typography>
-          </Stack>
-          <Button variant="contained" size="small" onClick={() => setOpenCreateDashboardDialogState(true)}>
-            Add Dashboard
-          </Button>
+    <Box>
+      <Stack direction="row" alignItems="center" justifyContent="space-between">
+        <Stack direction="row" alignItems="center" gap={1} my={2}>
+          <ViewDashboard />
+          <Typography variant="h3">Dashboards</Typography>
         </Stack>
-        <ErrorBoundary FallbackComponent={ErrorAlert}>
-          <DashboardList dashboardList={data} />
-        </ErrorBoundary>
-        <CreateDashboardDialog
-          open={openCreateDashboardDialogState}
-          onClose={() => setOpenCreateDashboardDialogState(false)}
-          onSuccess={(name: string) => handleDashboardCreation(name)}
-        />
-      </Box>
-    </Paper>
+        <Button variant="contained" size="small" onClick={() => setOpenCreateDashboardDialogState(true)}>
+          Add Dashboard
+        </Button>
+      </Stack>
+      <ErrorBoundary FallbackComponent={ErrorAlert}>
+        <Card>
+          <DashboardList dashboardList={data || []} hideToolbar={props.hideToolbar} />
+        </Card>
+      </ErrorBoundary>
+      <CreateDashboardDialog
+        open={openCreateDashboardDialogState}
+        onClose={() => setOpenCreateDashboardDialogState(false)}
+        onSuccess={(name: string) => handleDashboardCreation(name)}
+      />
+    </Box>
+  );
+}
+
+interface RecentVisitedDashboardsInProjectProperties {
+  projectName: string;
+}
+
+function RecentVisitedDashboardsInProject(props: RecentVisitedDashboardsInProjectProperties) {
```

Comment:
I like how you broke these out into components! Would it make sense to move the ones not called 'ViewProject' to: `ui/app/src/components`?

Or as this app grows, we could have additional folders within this views folder with sub components (like Luke [described here](https://github.com/perses/perses/pull/530#discussion_r973365075))

So something like: 
- /views/projects/RecentVisitedDashboardsInProject.tsx
- /views/projects/DashboardPageInProject.tsx

At that point, you could probably rename to `/views/projects/RecentlyVisitedDashboards.tsx` without the redundant "InProject" in the component name

Reply:
Another quick thought, this doesn't need to be in this PR or anything, but wanted to mention that Julie recently set up a lot of really cool testing infrastructure, you can see more info about e2e tests with Playwright [here](https://github.com/perses/perses/tree/main/ui/e2e) and I think she added the skeleton of some [initial tests](https://github.com/perses/perses/blob/main/ui/e2e/src/pages/AppProjectPage.ts) for these pages too

Reply:
I split as you told, is it good or we still want to keep `ViewProject` at the root of `View` directory?
I also renamed `ViewProject` to `ProjectView`, I think it's a better naming, wdyt? I will need to do another PR to update others View files after

Reply:
I think it'd be good to put ViewProject.tsx inside the projects folder too (sorry wasn't clear before) and then the components that are only used on the projects page could live there as well. Eventually, we'll also need `index.ts` barrel files to make importing from places like ui/app/src/components easier instead of the long relative imports (doesn't need to be done on this PR just thinking ahead). 

Also, if you have a different way to organize this that you prefer, just let me know! I'm definitely interested in new ideas